packages/code-studio/.env.development

```diff
@@ -1,7 +1,11 @@
 REACT_APP_ENABLE_LOG_PROXY=false
 REACT_APP_CORE_API_URL=http://localhost:10000/jsapi
-REACT_APP_NOTEBOOKS_URL=http://localhost:10000/notebooks
-REACT_APP_LAYOUTS_URL=http://localhost:10000/layouts
+
+# Proxy set in package.json to handle 4000 to solve CORS
+# https://create-react-app.dev/docs/proxying-api-requests-in-development/
+REACT_APP_NOTEBOOKS_URL=http://localhost:4000/notebooks
+REACT_APP_LAYOUTS_URL=http://localhost:4000/layouts
+
```

Comment:
You should be able to get rid of all of these lines, the defaults in `.env` actually work now with that proxy value set. You can leave a comment in this file that there's a proxy param in `package.json`, but yea we shouldn't need to specify any of these (eg. it defaults to /notebooks, since it's hosted on localhost:4000 it'll hit http://localhost:4000/notebooks)